### ISSUE TITLE
[Question] Is there a way to map more base type

### ISSUE THREAD

**Issue by mVanSteen** [WindhoverLabs/auto-yamcs#79]
Hello, while using auto-yamcs we stumbled upon a problem. While we use the same type as the core (`uint8`  or similar) there is absolutely no problem. But when we use `uint8_t` we have some issues. The type is recognized by juicer but there is no equivalence between both types that is created. Finally, in the XCTE file, uint8_t is not encoded as a `BaseType/uint8_BE` but just as `uint8_t`. This seems to confuse Yamcs which does not consider `uint8_t` as an integer but as an aggregate. 

Is there a way to fix this ? Is it a bug or did I just miss some configuration?

Thanks a lot for your help

**Comment by lorenzo-gomez-windhover** [comment]
> Is there a way to fix this ? Is it a bug or did I just miss some configuration?

Yes. https://github.com/WindhoverLabs/auto-yamcs/blob/develop/config/singleton_config.yaml#L1016
